BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -222,6 +222,15 @@
   # @Prompt Minimum length(in bytes) of the system preboot TCG event log area(LAML).
   gPlatformCommonLibTokenSpaceGuid.PcdTcgLogAreaMinLen|0x10000|UINT32|0x00010081
 
+[PcdsDynamic]
+  ## This PCD indicates the hash algorithm to be enabled by bootloader for measured boot
+  #  Based on the value set, PCR bank world be enabled and extended

```

Comment:
"This PCD indicates the PCR bank to be enabled/supported by Slim Bootloader for measured boot"

Reply:
Do we really need dynamic PCD ?  
We are trying to void if possible since it will increase the binary size.
Can this be a fixed config ?

Reply:
Platform configdata would define TPM PCR bank to be enabled. This dynamic pcd would be updated and used in TPMlib and necessary libs to check the PCR bank enabled. 
Incase if TPM Alg is NOT available in platform config data(for legacy platforms), default value in dynamic PDC would be used.